### PR TITLE
all methods were created, the model of manufacturer was created...

### PR DESCRIPTION
**Pull request by oleksiistepaniak**

### CODE REVIEW

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            Optional<Manufacturer> optionalManufacturer = null;
+            while (resultSet.next()) {
+                Long manufacturerId = resultSet.getObject(1, Long.class);
+                String name = resultSet.getString(COLUMN_NAME);
+                String country = resultSet.getString(COLUMN_COUNTRY);
+                Manufacturer manufacturer = new Manufacturer();
+                manufacturer.setId(manufacturerId);
+                manufacturer.setName(name);
+                manufacturer.setCountry(country);
+                optionalManufacturer = Optional.of(manufacturer);
```
Comment (karma-o) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:64]: code duplication, check common mistakes to fix that

**Review by mishok321 — APPROVED** [review]
good job

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteManufacturerRequest
+                = "UPDATE manufacturers SET is_deleted = true WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteManufacturerStatement = connection
+                        .prepareStatement(deleteManufacturerRequest,
+                               Statement.RETURN_GENERATED_KEYS)) {
```
Comment (karma-o) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:118]: do we generate any keys in the delete query?

**Review by karma-o — CHANGES_REQUESTED** [review]

**Review by nacenik — APPROVED** [review]
LGTM! 👍

**Pull request closed without merging**